content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
-```js
-// migrations/20191217102908-create-user.js
-module.exports = {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('Users', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER,
-      },
-      firstName: {
-        type: Sequelize.STRING,
-        field: 'first_name',
-      },
-      lastName: {
-        type: Sequelize.STRING,
-        field: 'last_name',
-      },
-      email: {
-        type: Sequelize.STRING,
-      },
-      emailConfirmed: {
-        type: Sequelize.BOOLEAN,
-        field: 'email_confirmed',
-        allowNull: false,
-        defaultValue: false,
-      },
-      birthDate: {
-        type: Sequelize.DATE,
-      },
-    })
-  },
-  down: (queryInterface, Sequelize) => {
-    return queryInterface.dropTable('Users')
-  },
+  @Column({ name: 'first_name' })
+  firstName: string
+
+  @Column({ name: 'last_name' })
+  lastName: string
+
+  @Column({ unique: true })
+  email: string
+
+|  @Column()
+|  salutation: string;
 }
 ```
 
-2. Create the corresponding model as in the previous section.
-3. To add a new table, create a new migration and corresponding model.
-   To change the `Users` table, create a new migration which will add the field and update the model to reflect the changes.
+Synchronizing such changes can be a challenge with ORMs because the changes are applied manually and are not easily verifiable programmatically. Renaming an existing column can be even more cumbersome and involve searching and replacing references to the column.
 
-The first two steps will be part of any schema change that uses Sequelize both as an ORM and for migrations. The second step is where the _synchronization culprit_ is. In other words, it is where the responsibility is on the developer to ensure she has correctly defined the model in line with the database schema.
+> **Note:** Django's [makemigrations](https://docs.djangoproject.com/en/3.1/ref/django-admin/#django-admin-makemigrations) CLI generates migrations by inspecting changes in models which, similar to Prisma, does away with the synchronization problem.
 
-Other ORMs have a workflow similar to this. Some, like [Django's ORM](https://docs.djangoproject.com/en/3.0/topics/migrations/#workflow), support a simpler workflow where you're required to only change the models and using a CLI tool, your models are scanned and compared to the versions currently contained in your migration files, and then a new set of migrations is written out.
+In summary, evolving the schema is a key part of building applications. With ORMs, the workflow for updating the schema involves using a migration tool to create a migration followed by updating the corresponding entity and mapper classes (depending on the implementation). As you'll see, Prisma takes a different approach to this.
 
 Now that you've seen migrations and workflows, you will look at the benefits and drawbacks of ORMs.
```

Comment:
"migrations and workflows" - what does "workflows" refer to?

Reply:
It was ambiguous as you pointed out. I've updated it to:

> Now that you've seen what migrations are and how they fit into the development workflows, you will learn more about the benefits and drawbacks of ORMs.